.golangci.yaml

```diff
@@ -0,0 +1,14 @@
+issues:
+  exclude-use-default: false
+  exclude-rules:
+    - text: "G107: Potential HTTP request made with variable url"
```

Comment:
good point. re-written as:
```yaml
exclude-rules:
	 - path: pkg/leader/http.go
      text: "G107: Potential HTTP request made with variable url"
      linters:
        - gosec
```
Bit more fragile though.

Reply:
That's not a lot better because most of these errors that the linter is pointing out will likely be in `http.go`, so it will still hide all instances rather than just the one (for now) that you want to hide.

It is possible to add `//nolint:gosec` to the line where the linter was pointing out the issue. It would be nice if you could specify exactly which error to ignore (G107) in the comment, but there are no examples of that in the golangci-lint README, and that's the only documentation.

But I think adding that comment in the source is better as it explains exactly what is being targeted by the exclusion.

Reply:
ok - i've removed everything and i get no linter warnings. weird.
sometimes it feels like the linter is inconsistent.
it was an error when i had the url as a global var in main.go.... who knows.